recipes/perceptualdiff/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "perceptualdiff" %}
+{% set version = "2.1" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+source:
+    url: https://github.com/myint/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 0dea51046601e4d23dc45a3ec342f1a305baf3bf3328e9ccdae115fe1942f041
+build:
+    number: 0
+    skip: true  # [win]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - llvm-openmp  # [osx]
+        - libgomp  # [linux]
+        - cmake
+        - make
+        - freeimage
```

Comment:
```suggestion
```

Reply:
Thanks, due to this correct?

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host